### PR TITLE
QPPA-4422 fix qcdr stratum 2020

### PR DESCRIPTION
**Pull request by dschraitle**
#### Motivation for change

Fixes stratum descriptions for 2020 QCDR measures

#### What is being changed

Build script relies on the format "Rate \d:" in the measure description to determine the strata descriptions. A previous change had removed this format from several measures, this change puts them back.

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [x] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [ ] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

##### Associated JIRA tickets:
* https://jira.cms.gov/browse/QPPA-4422

### CODE REVIEW

**Comment by bartcone** [comment]
@dschraitle go ahead and bump the minor on this PR.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "3.1.4",
+  "version": "3.2.0",
```
Comment (branonbarrett) [inline package.json:3]: Please revert this. package.json bump should happen on the release branch

**Comment by branonbarrett** [comment]
> @dschraitle go ahead and bump the minor on this PR.

@bartcone we should be bumping this on the release branch in case there are multiple updates occurring in a given release

**Comment by bartcone** [comment]
> @bartcone we should be bumping this on the release branch in case there are multiple updates occurring in a given release
@branonbarrett yeah - I didn't see that this repo had switched over to a gitflow style. My bad @dschraitle !

**Pull request merged**